src/client/stdlib/inputs.js

```diff
@@ -1,5 +1,60 @@
-import {fileOf} from "@observablehq/inputs";
+import {file as _file} from "@observablehq/inputs";
 import {AbstractFile} from "npm:@observablehq/stdlib";
 
-export * from "@observablehq/inputs";
-export const file = fileOf(AbstractFile);
+export {
+  button,
+  checkbox,
+  radio,
+  toggle,
+  color,
+  date,
+  datetime,
+  form,
+  range,
+  number,
+  search,
+  searchFilter,
+  select,
+  table,
+  text,
+  email,
+  tel,
+  url,
+  password,
+  textarea,
+  input,
+  bind,
+  disposal,
+  formatDate,
+  formatLocaleAuto,
+  formatLocaleNumber,
+  formatTrim,
+  formatAuto,
+  formatNumber
+} from "@observablehq/inputs";
```

Comment:
Repeating this is unfortunate but ESLint doesn’t like us exporting `file` twice. 😞